packages/oidc-auth/src/index.test.ts

```diff
@@ -374,6 +375,27 @@ describe('oidcAuthMiddleware()', () => {
     expect(res.status).toBe(302)
     expect(res.headers.get('set-cookie')).toMatch(`Domain=${MOCK_COOKIE_DOMAIN}`)
   })
+  test('Should use custom audience if defined', async () => {
+    process.env.OIDC_AUDIENCE = 'audience'
+    const req = new Request('http://localhost/', {
+      method: 'GET',
+      headers: { cookie: `oidc-auth=${MOCK_JWT_EXPIRED_SESSION}` },
+    })
+    const res = await app.request(req, {}, {})
+    expect(res).not.toBeNull()
+    expect(res.status).toBe(302)
+    expect(res.headers.get('location')).toMatch(/audience=audience/)
+  })
+  test('Should not set audience if not defined', async () => {
+    const req = new Request('http://localhost/', {
+      method: 'GET',
+      headers: { cookie: `oidc-auth=${MOCK_JWT_EXPIRED_SESSION}` },
+    })
+    const res = await app.request(req, {}, {})
+    expect(res).not.toBeNull()
+    expect(res.status).toBe(302)
+    expect(res.headers.get('location')).not.toMatch(/audience=audience/)
```

Comment:
Maybe a search for just the url param would be a more general assert?

```suggestion
    expect(res.headers.get('location')).not.toMatch(/audience=/)
```

Reply:
Sure, that makes sense. Updated also.